checkov/secrets/runner.py

```diff
@@ -225,6 +225,8 @@ def run(
             secret_records: dict[str, SecretsRecord] = {}
             secrets_in_uuid_form = ['CKV_SECRET_116']
             for key, secret in secrets:
+                if secret.secret_value and 'vault:' in secret.secret_value.lower() and secret.check_id in ENTROPY_CHECK_IDS:
```

Comment:
extract all those if into an external function ;)

Reply:
+ add debug log that we filtered it
+ Maybe its better to put this after (current) line 233, where we have the check_id either from the secret or from SECRET_TYPE_TO_ID?